validator/cpp/htmlparser/parser_fuzz.cc

```diff
@@ -0,0 +1,11 @@
+#include "cpp/htmlparser/parser.h"
+
+#include "fuzztest/fuzztest.h"
+
+// Fuzz the parsing logic. Any non-crash is a success for the fuzzer.
+void FuzzParser(std::string raw_html) {
+  htmlparser::Parser parser(raw_html);
+  parser.Parse();
+}
+
+FUZZ_TEST(CC_FUZZING, FuzzParser);
```

Comment:
Consider using a more descriptive test suite name (e.g., the one used in validator/cpp/htmlparser/parser_test.cc) and a property function name that describes the tested property, e.g.:

`FUZZ_TEST(ParserTest, ParserNeverCrashes);`